apps/vr-tests/package.json

```diff
@@ -27,6 +27,7 @@
     "@fluentui/react-hooks": "*",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28634&commitId=2067ff29dda7d035846688c2657938a35d37feeb&env=prod&clientType=FLUENTUI)


  <details><summary><b>PieChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PieChart.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>